### PR TITLE
Fixed some things related to localization

### PR DESCRIPTION
**Pull request by oscargus**
Fixed a number of TODOs/FIXMEs.

### CODE REVIEW

File: src/main/java/net/sf/jabref/external/AttachFileAction.java [hunk 1]
```diff
@@ -25,7 +26,8 @@ public AttachFileAction(BasePanel panel) {
     @Override
     public void action() {
         if (panel.getSelectedEntries().size() != 1) {
-            return; // TODO: display error message?
+            panel.output(Localization.lang("No entries or multiple entries selected."));
```
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/external/AttachFileAction.java]: Add that the action was aborted.

File: src/main/java/net/sf/jabref/external/FindFullTextAction.java [hunk 2]
```diff
@@ -93,7 +95,7 @@ public void update() {
             basePanel.output(Localization.lang("Finished downloading full text document"));
         }
         else {
-            String message = Localization.lang("Full text article download failed");
+            String message = Localization.lang("Full text document download failed");
```
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/external/FindFullTextAction.java]: just remove `document`?

**Comment by tobiasdiez** [comment]
LGTM :+1:

File: src/main/java/net/sf/jabref/external/AttachFileAction.java (hunk 1 above)
Comment (oscargus) [inline src/main/java/net/sf/jabref/external/AttachFileAction.java]: We should look over that in general. The next action (in the BasePanel (?) code) has exactly the same message. There are also a few related translations strings. For example ~"You must select exactly one entry".

Also, one can consider if a dialog should be shown, possibly with a "do not show this again check box", for this and many other issues. Drawback is that the check box selection must be stored somehow, but maybe one can create a dedicated dialog type for that?

File: src/main/java/net/sf/jabref/external/FindFullTextAction.java (hunk 2 above)
Comment (oscargus) [inline src/main/java/net/sf/jabref/external/FindFullTextAction.java]: For consistency I think document should be there (and that document is the correct word). Many of the translations already use document by the way.

File: src/main/java/net/sf/jabref/external/AttachFileAction.java (hunk 1 above)
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/external/AttachFileAction.java]: I would prefer if you could clean up all the related messages as well in this PR. Right now the output is just a statement and not an error message.

File: src/main/java/net/sf/jabref/external/FindFullTextAction.java (hunk 2 above)
Comment (oscargus) [inline src/main/java/net/sf/jabref/external/FindFullTextAction.java]: I changed this text to "This operation requires exactly one item to be selected." (new string) and also replaced "No entries selected." with "This operation requires one or more entries to be selected." (already existing).

**Comment by simonharrer** [comment]
Are you done? @oscargus :) Stefan and myself are currently looking at the pull requests and trying to reduce the huge amount (>30) by merging them when suitable.

**Comment by oscargus** [comment]
Now I think I'm done. :-) If nothing else I have teaching now...

**Pull request merged**